app/src/main/java/com/rkpandey/mymaps/CreateMapActivity.kt

```diff
@@ -116,5 +120,26 @@ class CreateMapActivity : AppCompatActivity(), OnMapReadyCallback {
             markers.add(marker)
             dialog.dismiss()
         }
+        val spinner = dialog.findViewById<Spinner>(R.id.crimeSpinner)
+        spinner?.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
```

Comment:
using kotlin view extensions was also crashing the app but just using a find view by id fixed that 🤷

Reply:
That's really strange.